proposals/PROPOSALS.md

```diff
@@ -1,10 +1,29 @@
-An HFTP (*Haskell Foundation Technical Proposal*) is a process for submitting a proposal for a technical initiative to that will be executed under the guidance of the Haskell Foundation (HF). This document describes how the community may propose, discuss, and implement technical initiatives to be carried out through the Haskell Foundation that will affect the Haskell ecosystem. Each HFTP goes through an iterative process in which it is discussed by the HF Technical Track (HFTT), the broader community, and any and all interested stakeholders. If a consensus is reached, the initiative is accepted and may then be executed by the Foundation (resources permitting). Only upon reaching a consensus are initiatives accepted and executed. Upon acceptance, an HFTP and its associated discussion are merged into this repository, and both serve as a historical document that details a specification and rationale for work detailed within the process.
+An HFTP (*Haskell Foundation Technical Proposal*) is a proposal for a technical initiative that will be executed under the guidance of the Haskell Foundation (HF). Each HFTP goes through an iterative process in which it is discussed by the HF Technical Working Group (TWG), the broader community, and any and all interested stakeholders. As part of this process, resources are identified that could ease or make possible the implementation of the proposal.
 
 The aim of the HFTP process is to apply the openness and collaboration that have shaped Haskell's documentation and implementation to a process of evolving the HF and the broader Haskell ecosystem. This document captures our guidelines, commitments and expectations regarding this process.
 
+## Acceptance
+
+Upon acceptance, an HFTP and its associated discussion are merged into this repository, and both serve as a historical document that details a specification and rationale for work detailed within the process. Acceptance of a proposal does not constitute a firm commitment by the Haskell Foundation to execute on the proposal, or to provide the necessary or helpful resources that were identified in the proposal. Fundamentally, acceptance indicates that the TWG believes that the Haskell ecosystem would be better off if the proposal were implemented, and that it is sufficiently well-considered.
+
+That said, the Haskell Foundation is keenly interested in making the Haskell ecosystem better off. Within the constraints of the Foundation's resources and strategic plans, accepted proposals will be considered as good candidates for tasks to be executed on.
+
+
+## Submission process summary
+
+1. Submit a Pre-HFTP topic to [Discourse](https://discourse.haskell.org), soliciting informal feedback.
```

Comment:
I would really like to say that the author is free to fork the repo and create a proposal, but not make a PR, in order to provide a document for the pre-HFTP discussion to look at.   I.e. as part of step 1.   Yes, someone could make a Google doc and socialise that, and then transfer it to the repo in step 2, but that is just bureaucratic busy-work.

Reply:
I actually would prefer that discourse posts be more of the character of "I'd like to build a WIDGET that will help Haskell in these ways. Has anyone else made the WIDGET? Am I missing something important?" rather than "Here's my long formal document that I've put lots of time into. Please comment both here and in the repo, and let's scatter our discussion."

Basically, I think that if you've already put in the time to write a long detailed document, then you're past the point where the Discourse discussion has the most potential to be helpful. Doubly so for people not as well-connected and comfortable with long-form formal writing as you.

What do you think?

Reply:
If you are saying "It's a formal requirement that you solicit feedback based on a 3-line summary to Discourse" then fine -- but I'm not sure it's worth formalising.

I would rather we do what we do on the GHC Proposals process. 
* The author creates a proposal, in the repo, and invites feedback from the commuinty
* A sometimes-extensive discussion follows, typically with multiple revisions of the proposal.  The committee are free to contribute, but have no obligation to do so.
* Finally the author thinks they are done, and submits the refined proposal to the committee.  At that point the committee has an obligation to get involved.

At every stage there is a proposal to discuss, which helps to focus the discussion to be about what the author is acutally proposing, rather than about what the audience fantasises that they might be proposing.

A sensible author will not invest a huge amount of time in a proposal and only then share it and solicit feedback.   More typcially they'll start with a sketch.  But that's up to them.   They are *free* to do a big up-front investment if they want.

The key goal is that by the time it gets to the committee, the wider community has had ample chance to discuss, reivew, say "oh I did that last year" or whatever.

It would be fine to require a post to Discourse advertising the work.  But I think it should be fine for the post to say "I've written this sketch proposal, please comment there" or "I've written this detailed proposal, please comment there".  I'm just asking please let's not prevent or discourage authors from writing as detailed a proposal as they please in the same format as they will ultimately submit it, as a way to generate and focus the discussion they want.